packages/experiments/src/components/fluent/theme/FluentColors.ts

```diff
@@ -0,0 +1,245 @@
+/* tslint:disable:no-any */
+export const GrayColors: any = {
```

Comment:
Does out linter allow type 'any'?  In any case, maybe `[key: string]: string` would be better.. marginally